### PR TITLE
Refactoring newRelicHandle signature

### PR DESCRIPTION
**Pull request by vidalvasconcelos**
This PR fix some comment typos and refactoring `newRelicHandle` function signature, using the type interfaces from `net/http` instead of closure signature.

### CODE REVIEW

File: api/new_relic.go [hunk 1]
```diff
@@ -23,9 +23,10 @@ func newRelicApp() *newrelic.Application {
 	return app
 }
 
-func newRelicHandle(app *newrelic.Application, pth string, f func(http.ResponseWriter, *http.Request)) (string, func(http.ResponseWriter, *http.Request)) {
+func newRelicHandle(app *newrelic.Application, pth string, f http.HandlerFunc) (string, http.HandlerFunc) {
 	if app == nil {
 		return pth, f
 	}
+
```
Comment (cuducos) [inline api/new_relic.go:30]: Just out of curiosity: is there a reason to add that blanl line?

**Comment by cuducos** [comment]
Thank you for the typos fixes, and for the enhancement in the signature — way better, right?

Next time, please, open two PRs: one for each change. It's easier and quick to review and merge this way ; )

File: api/new_relic.go (hunk 1 above)
Comment (vidalvasconcelos) [reply api/new_relic.go:30]: I imagine I added it by accident :|

**Comment by vidalvasconcelos** [comment]
I can split the fixes in a new PR, sorry my bad!

**Comment by cuducos** [comment]
> I can split the fixes in a new PR, sorry my bad!

No, no need now. Just keep it in mind next time : )

File: api/new_relic.go (hunk 1 above)
Comment (cuducos) [reply api/new_relic.go:30]: NP

I just asked to make sure I wasn't missing anything : )

**Review by cuducos — APPROVED** [review]

**Pull request merged**